src/reducers/measurements/types.ts

```diff
@@ -0,0 +1,100 @@
+import { measurementIdSymbol } from "../../util/globals";
+
+// -- Measurements JSON types -- //
+
+/**
+ * Measurements are allowed to have arbitrary metadata.
+ * Matching types allowed in Augur's measurements schema
+ * <https://github.com/nextstrain/augur/blob/3f72c40897a80132099729d5d00a6718e76e0e9e/augur/data/schema-measurements.json#L152>
+ */
+type MeasurementMetadata = string | number | boolean
+
+export const measurementsDisplayValues = ["raw", "mean"];
+export type MeasurementsDisplay = (typeof measurementsDisplayValues)[number]
+export const isMeasurementsDisplay = (x: any): x is MeasurementsDisplay => measurementsDisplayValues.includes(x);
```

Comment:
Specifying the values again is redundant but this seems more readable:


```suggestion
export const measurementsDisplayValues: MeasurementsDisplay[] = ["raw", "mean"];
export type MeasurementsDisplay = "raw" | "mean";
```

Reply:
Sure! I was briefly worried about values falling out of sync, but these shouldn't get updated. 
(Will do the same for `queryBooleanValues`/`QueryBoolean`)

Reply:
Updated in https://github.com/nextstrain/auspice/pull/1910/commits/262149e9a2eb041ab9c25ebe5635d4405a365ad2